stix_shifter_modules/async_dummy/stix_transmission/async_dummy_api_client.py

```diff
@@ -21,24 +21,21 @@ def __init__(self, connection, configuration):
 
     def ping_data_source(self):
         # Pings the data source
-        return "async ping"
+        return {"code": 200, "success": True}
 
     def create_search(self, query_expression):
         # Queries the data source
-        return {
-            "code": 200,
-            "query_id": "uuid_1234567890"
-        }
+        return {"code": 200, "query_id": "uuid_1234567890"}
```

Comment:
should it be `"search_id"` instead of `"query_id"` ?

Reply:
right, 

$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'success': True, 'search_id': 'query1'}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' status 'query1'
{'success': True, 'status': 'COMPLETED', 'progress': 100}
$ 
$ python main.py transmit synchronous_dummy '{"conn": "True"}' '{"conf": "True"}' results 'query1' 0 1
{'success': True, 'data': 'Results from search'}
$ 
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' ping
{'success': True}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' query 'query1'
{'search_id': 'uuid_1234567890'}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' status 'uuid_1234567890'
{'success': True, 'status': 'COMPLETED'}
$ 
$ python main.py transmit async_dummy '{"conn": "True"}' '{"conf": "True"}' results 'uuid_1234567890' 0 1
{'success': True, 'data': 'Results from search'}